src/main/java/module/youth/YouthTrainingView.java

```diff
@@ -68,4 +73,10 @@ public void reInit() {
     public void storeUserSettings() {
         this.tableModel.storeUserSettings(table);
     }
+/*
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        var youthPanel = (YouthPanel) HOMainFrame.instance().getTabbedPane().getModulePanel(IModule.YOUTH);
+        youthPanel.refreshYouthPlayerView();
+    }*/
```

Comment:
Is this needed since this is commented out?

Reply:
no it is not needed anymore - removed it.